browser/src/app/TilesMiddleware.ts

```diff
@@ -47,7 +47,7 @@ class TileCoordData {
 		this.part = part !== null ? part : app.map._docLayer._selectedPart;
 		this.mode = mode !== undefined ? mode : 0;
 
-		this.scale = Math.round(Math.pow(1.2, this.z - 10) * 1000) / 1000;
+		this.scale = Math.round(Math.pow(1.2, this.z - 10) * 10000) / 10000;
```

Comment:
I'm dubious as to why this was here in the first place. Personally, I'd rather the opposite change - remove the squashing of the decimals here and add a comparison utility function when we only need to know that something approximately equals this. This feels like it'd end up causing all sorts of oddities with regards to accuracy that would be a pain to track down if you weren't aware this happened.